File: python/small programs/recursion/fibonacci/fibonacci.py
def fib(n):
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n-1) + fib(n-2)


def suffix(n):
    if input % 100 == 11 or input % 100 == 12 or input % 100 == 13:
        suffix = "th"
    elif input % 10 == 1:
        suffix = "st"
    elif input % 10 == 2:
        suffix = "nd"
    elif input % 10 == 3:
        suffix = "rd"
    else:
        suffix = "th"
    return suffix

input = int(input("Enter an integer: "))
fib = fib(input)
print(f"{input}{suffix(input)} number in Fibonacci sequence is {fib}.")

File: java/small_programs/Fibonacci/src/Fibonacci.java
/* The program returns a list of n numbers in Fibonacci sequence.
   n is specified by the user.
 */

import java.util.Scanner;

public class Fibonacci {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter an integer (which will be the n-th number in Fibonacci sequence): ");
        int input = scanner.nextInt();
        for (int i = 1; i <= input; i++) {
            System.out.println("For number " + i + " the sum of Fibonacci sequence is: " + fibonacci(i));
        }
    }

    public static int fibonacci(int num) {
        if (num == 1 || num == 2) {
            return 1;
        }
        return num = fibonacci(num - 1) + fibonacci(num - 2);
    }
}
